internal/pkg/build/remotebuilder/remotebuilder.go

```diff
@@ -84,159 +80,87 @@ func (rb *RemoteBuilder) Build(ctx context.Context) (err error) {
 		libraryRef = rb.ImagePath
 	}
 
-	// Send build request to Remote Build Service
-	rd, err := rb.doBuildRequest(ctx, rb.Definition, libraryRef)
+	if libraryRef != "" && !client.IsLibraryPushRef(libraryRef) {
+		err = fmt.Errorf("invalid library reference: %v", rb.ImagePath)
+		sylog.Warningf("%v", err)
+		return err
+	}
+
+	br := buildclient.BuildRequest{
+		LibraryRef:          libraryRef,
+		LibraryURL:          rb.LibraryURL,
+		DefinitionRaw:       rb.Definition.Raw,
+		BuilderRequirements: rb.BuilderRequirements,
+	}
+
+	bi, err := rb.BuildClient.Submit(ctx, br)
 	if err != nil {
 		err = errors.Wrap(err, "failed to post request to remote build service")
 		sylog.Warningf("%v", err)
```

Comment:
Does this need to print the `err`? Its still returning it.

Reply:
Was just keeping current behavior in place, not something I was trying to change in this PR.